addons/binding/org.openhab.binding.boschindego/deindego.map

```diff
@@ -0,0 +1,23 @@
+0=Status lesen
```

Comment:
Both UIs should show the texts instead of the numbers. If they currently do not, I suspect that this is due to the bug https://github.com/eclipse/smarthome/issues/1550.
I'd therefore vote for keeping the binding clean and remove the map files and rather try to fix that bug soon.